conductor-client/src/main/kotlin/com/openlattice/hazelcast/serializers/OrganizationStreamSerializer.kt

```diff
@@ -65,11 +68,14 @@ class OrganizationStreamSerializer : SelfRegisteringStreamSerializer<Organizatio
             val connections = SetStreamSerializers.fastStringSetDeserialize(input)
 
             val grants = deserializeMapMap(input,
-                    mutableMapOf(),
-                    { UUIDStreamSerializerUtils.deserialize(input) },
-                    { GrantTypeStreamSerializer.deserialize(input) },
-                    { GrantStreamSerializer.deserialize(input) })
+                                           mutableMapOf(),
+                                           { UUIDStreamSerializerUtils.deserialize(input) },
+                                           { GrantTypeStreamSerializer.deserialize(input) },
+                                           { GrantStreamSerializer.deserialize(input) })
```

Comment:
So much indentation... is this really what our formatting styles want?

Reply:
Yeah. I think the latest Kotlin plugin updated formatting guidelines and it might be what is getting applied here.